From e7199d92ccb7699f47155feeaf3977d177374189 Mon Sep 17 00:00:00 2001
From: michaelsh <michaelsh@marvell.com>
Date: Sun, 13 Jan 2019 17:08:57 +0200
Subject: [PATCH 22/30] examples: ipsec-secgw: Added <pri> field in ep1.cfg
 file

Added <pri> field in ep1.cfg file in records where the field was
absent. Without the field application fails with bad configuration
error. First configuration file ep0.cfg has this <pri> field
and applicaiton runs correctly if used ep0.cfg.

Change-Id: I611cdd6b87d9b71137ea7cf7bbcadd971c2099c5
Signed-off-by: michaelsh <michaelsh@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/2047
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Narayana Prasad Raju Athreya <Prasad.Athreya@cavium.com>
Reviewed-by: Alan Winkowski
---
 examples/ipsec-secgw/ep1.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/examples/ipsec-secgw/ep1.cfg b/examples/ipsec-secgw/ep1.cfg
index 3f6ff81..6bf4d37 100644
--- a/examples/ipsec-secgw/ep1.cfg
+++ b/examples/ipsec-secgw/ep1.cfg
@@ -19,8 +19,8 @@ sp ipv4 in esp protect 15 pri 1 dst 192.168.200.0/24 sport 0:65535 dport 0:65535
 sp ipv4 in esp protect 16 pri 1 dst 192.168.201.0/24 sport 0:65535 dport 0:65535
 sp ipv4 in esp protect 25 pri 1 dst 192.168.55.0/24 sport 0:65535 dport 0:65535
 sp ipv4 in esp protect 26 pri 1 dst 192.168.56.0/24 sport 0:65535 dport 0:65535
-sp ipv4 in esp bypass dst 192.168.240.0/24 sport 0:65535 dport 0:65535
-sp ipv4 in esp bypass dst 192.168.241.0/24 sport 0:65535 dport 0:65535
+sp ipv4 in esp bypass pri 1 dst 192.168.240.0/24 sport 0:65535 dport 0:65535
+sp ipv4 in esp bypass pri 1 dst 192.168.241.0/24 sport 0:65535 dport 0:65535
 
 sp ipv4 out esp protect 105 pri 1 dst 192.168.115.0/24 sport 0:65535 dport 0:65535
 sp ipv4 out esp protect 106 pri 1 dst 192.168.116.0/24 sport 0:65535 dport 0:65535
-- 
2.7.4

